lib/bundle/brew_dumper.rb

```diff
@@ -75,6 +82,7 @@ def formula_inspector(f)
       {
         :name => f["name"],
         :full_name => f["full_name"],
+        :aliases => f["aliases"],
```

Comment:
It can cause some problem. For example, `objective-caml` is not an alias but an old name for `ocaml`. 

Maybe we can simplify this. Instead of supporting old name. We create alias explicitly in such case, since it will never be used by another formula.